tasks.py

```diff
@@ -0,0 +1,77 @@
+### Using the element path strategy when locating application elements.
+
+import io
+import random
+
+from robocorp.tasks import task, setup
+from robocorp.tasks._protocols import ITask
+from robocorp import log
+from robocorp import windows
+
+APP_EXECUTABLE = "calc.exe"
+APP_LOCATOR = "name:Calculator"
+PATH_TO_MAINVIEW = "path:2|3|2"
+PATH_TO_RESULT = f"{PATH_TO_MAINVIEW}|2|1" # path:2|3|2|2|1
+# Different element sections
+PATH_TO_CONTROLS = f"{PATH_TO_MAINVIEW}|5" # path:2|3|2|5
+PATH_TO_OPERATORS = f"{PATH_TO_MAINVIEW}|7" # path:2|3|2|7
+PATH_TO_NUMBERPAD = f"{PATH_TO_MAINVIEW}|8" # path:2|3|2|8
+# Clear button in Controls section
+PATH_TO_CLEAR_BUTTON = f"{PATH_TO_CONTROLS}|3" # path:2|3|2|5|3
+# Buttons in Operators section
+PATH_TO_MINUS_BUTTON = f"{PATH_TO_OPERATORS}|3" # path:2|3|2|7|3
+PATH_TO_PLUS_BUTTON = f"{PATH_TO_OPERATORS}|4" # path:2|3|2|7|4
+PATH_TO_EQUALS_BUTTON = f"{PATH_TO_OPERATORS}|5" # path:2|3|2|7|5
+PATH_TO_TO_NUMBER_FIVE = "path:2|3|2 > id:num5Button"
+
+RANDOM_INT_RANGE = range(1, 9)
```

Comment:
This is a caveat. `range` returns an iterator, so, it'll only iterate correctly once. If you want to iterate multiple times you need to do:

`RANDOM_INT_RANGE = tuple(range(1, 9))`

But to be fair, I wouldn't even create a constant to this one.